santa-tracker-base.conf

```diff
@@ -1,92 +1,67 @@
 # Performance scenarios for santa-tracker-android project
 
-file-to-change = "santa-tracker/src/main/java/com/google/android/apps/santatracker/customviews/SnowFlake.java"
-resource-to-change = "santa-tracker/src/main/res/values/strings.xml"
+# These are set in santa-tracker-kotlin.scenarios or santa-tracker-java.scenarios
+file-to-change = ""
+resource-to-change = ""
 
 base {
     gradle-args = [
         "-Dorg.gradle.workers.max=8",
         "--init-script", "../scans.init.gradle",
+        "--no-scan",
+        "--no-build-cache",
     ]
-    warm-ups = 4
-}
 
-configurationTime = ${base} {
-    tasks = ["help"]
-    warm-ups = 10
-}
+    warm-ups = 1
```

Comment:
I would leave out setting the warm-ups in the base to `1`. I think the default is as good as well.

Reply:
👍